sound/soc/sof/intel/hda-loader.c

```diff
@@ -385,11 +385,6 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
 	if (i == HDA_FW_BOOT_ATTEMPTS) {
 		dev_err(sdev->dev, "error: dsp init failed after %d attempts with err: %d\n",
 			i, ret);
-		dev_err(sdev->dev, "ROM error=0x%x: FW status=0x%x\n",
-			snd_sof_dsp_read(sdev, HDA_DSP_BAR,
-					 HDA_DSP_SRAM_REG_ROM_ERROR),
```

Comment:
for my education, where is HDA_DSP_SRAM_REG_ROM_ERROR read? I see a ROM_STATUS but not ROM_ERROR in the rest of the code?

Reply:
in hda_dsp_dump_ext_rom_status(). the ROM error would be the second dword in the extended status.

Reply:
ok